taskfiles/ci.yml

```diff
@@ -134,6 +134,9 @@ tasks:
       - 'echo "~~~ Logging into Dockerhub :docker:"'
       - cmd: docker login --username {{.DOCKERHUB_USER}} --password {{.DOCKERHUB_TOKEN}}
         silent: true
+      - 'echo "~~~ Logging into ECR :ecr:"'
+      - aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws/l9j0i2e0
```

Comment:
I'm not sure if docker logout should consist both registry (line 140 in changed file).